diff --git a/node_modules/@pawan-pk/react-native-mapbox-navigation/android/build.gradle b/node_modules/@pawan-pk/react-native-mapbox-navigation/android/build.gradle
index 7d28df3..3656ea3 100644
--- a/node_modules/@pawan-pk/react-native-mapbox-navigation/android/build.gradle
+++ b/node_modules/@pawan-pk/react-native-mapbox-navigation/android/build.gradle
@@ -116,9 +116,9 @@ dependencies {
   implementation "com.facebook.react:react-native:+"
   implementation "org.jetbrains.kotlin:kotlin-stdlib:$kotlin_version"
 
-  // mapbox dependencies
-  implementation "com.mapbox.navigationcore:navigation:3.0.2"
-  implementation "com.mapbox.navigationcore:ui-components:3.0.2"
+  // Mapbox Navigation SDK v3.18.0
+  implementation "com.mapbox.navigationcore:navigation:3.18.0"
+  implementation "com.mapbox.navigationcore:ui-components:3.18.0"
 
   implementation 'androidx.constraintlayout:constraintlayout:2.1.4'
   implementation 'androidx.cardview:cardview:1.0.0'
